odysseus/odysseus_tree/overlays/rootfs_overlay_nrc_common/usr/bin/build_nrc_params.py

```diff
@@ -12,12 +13,15 @@
 
 '''
 
+parser = argparse.ArgumentParser()
+
+parser.add_argument('option', choices=['STA', 'AP'], help="STA or AP")
+parser.add_argument('--ip', help="path/to/this/ini/file (optional, otherwise default: /etc/nrc_opts_<ap|sta>.ini)")
```

Comment:
can you make them posix compliant.
This means two long works and a short path with a single dash as well,
so `--ini-path` and `-i`
Example from docs:
`parser.add_argument('-c', '--count')      # option that takes a value`

Reply:
you dont have to do that for the option field just the other two.